apps/site/pages/en/blog/announcements/official-discord-launch-announcement.md

```diff
@@ -0,0 +1,42 @@
+---
+date: '2025-02-11T21:00:00.000Z'
+category: announcements
+title: Node.js Launches Official Community Space on Discord
+layout: blog-post
+author: Carl Vitullo, Claudio Wunder
+---
+
+We are thrilled to announce the launch of the official Node.js community space on Discord! Over the past few months, in collaboration with the [Reactiflux](https://www.reactiflux.com/) community, we have been working diligently to create a vibrant and engaging space for Node.js enthusiasts. Today, we are excited to invite you to join us on this new platform!
+
+### Why Discord?
+
+[Discord](https://discord.com) has rapidly become a go-to platform for communities to connect and communicate. Many Node.js community members already use Discord to discuss Node.js, seek advice, and share their projects. By establishing an official Node.js Discord server, we aim to centralize these conversations and provide a dedicated space for our community. Additionally, numerous other open-source projects, such as [TypeScript](https://discord.gg/typescript), [Rust](https://discord.gg/rust-lang), and [Kubernetes](https://discord.gg/kubernetes), have successfully built their communities on Discord.
+
+Our Node.js Discord space will be perfect for:
+
+- Hosting livestreams by [Node.js Ambassadors](https://github.com/nodejs/nodejs-ambassadors) and community members
```

Comment:
This link is 404 for me, perhaps current a private repo?

Reply:
Hmmm, good point; Seems private.